components/Member/Profile/Header.js

```diff
@@ -2,11 +2,28 @@ import React from 'react';
 import styled from 'styled-components';
 
 import Stepper from './Stepper';
+import { above, below } from '../../../utilities';
```

Comment:
@kburnell I moved a bunch of these around, and still may refactor some. Let me know how I can help align this to what you are cooking up.

Reply:
a bunch of what @saragibby ?

Reply:
moved the member pages and components around, and no promises I'm done yet 🙃

Reply:
That won't impact me at all.  I am not doing anything with them